tests/python/unittest/test_meta_schedule_tune_relay.py

```diff
@@ -323,6 +326,227 @@ def get_output(data, lib):
         assert np.allclose(actual_output, expected_output, rtol=1e-4, atol=2e-4)
 
 
+@register
+def int32x16(imm, span):
```

Comment:
shall we move this registration logic to TVMScript parser to formally support it? Thanks a lot!

Reply:
done